### PR TITLE
Add rudimentary HAT link to send-money section

### PR DESCRIPTION
**Pull request by codewhisperer**
Quick and dirty, to get some data and awareness on HAT interest

image

### CODE REVIEW

File: items/navigation.json [hunk 1]
```diff
@@ -10,6 +10,10 @@
         "link": "/"
       },
       "items": [
+        {
+          "translationKey": "public-navigation.money-transfer.sendHighAmount",
```
Comment (mikknarusk) [inline items/navigation.json]: let's keep to one style of keys, this repo uses kebab-case

File: items/navigation.json [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+          "link": "/?sourceAmount=1000000&sourceCurrency=GBP&targetCurrency=USD"
```
Comment (KarlEST) [inline items/navigation.json]: do we want to change customer selected source and target currency?

Comment (codewhisperer) [reply items/navigation.json]: That way I don't have to add the logic of what amounts to a high value in which currency. That's what I meant by quick and dirty

Comment (codewhisperer) [reply items/navigation.json]: I think if I needed to add that logic it should probs sit in homepage

**Pull request merged**